msrcrypto.js

```diff
@@ -15,7 +15,9 @@
 //    limitations under the License.
 //
 //*******************************************************************************
-if (typeof module !== 'undefined') {
+var nodeJsSupport = (typeof module !== 'undefined') && (this.module !== module)

```

Comment:
Can these changes go in our repo instead of MSR? this would create the need to rebase/merge on every msft update